### ISSUE TITLE
Unable to Compile Due to Missing v4l2-uvc.h Header

### ISSUE THREAD

**Issue by WagnerNasc** [rexionmars/Nitro5-AN515-58-WebCam-Module#2]
When compiling using the command "**make -j4 -C /lib/modules/$(uname -r)/build M=$(pwd) modules**" it is generating the following error:

```
/home/wagnasla/camera_fix/linux-hwe-6.5-6.5.0/drivers/media/usb/uvc/uvc_driver.c:24:10: fatal error: media/v4l2-uvc.h: No such file or directory
   24 | #include <media/v4l2-uvc.h>

```

**My OS configuration:**

Operating System: **Ubuntu 22.04.4 LTS**
Kernel Version: **6.5.0-18-generic**
Compiler Version: **gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0**

Thank you for your assistance in resolving this issue.

**Comment by rexionmars** [comment]
@WagnerNasc 
Hello, I apologize for the delay. I've just updated the instructions to fix the issue using Ubuntu 22.04 with kernel 6.5.
https://github.com/rexionmars/Nitro5-AN515-58-WebCam-Module/blob/master/README.md

**Comment by rexionmars** [comment]
Video: https://youtu.be/NDKrSZ1Tkoo?si=o7-YQcPoaQJZldkT
